install/kubernetes/helm/Istio/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: Istio
+version: 0.1.0-1
```

Comment:
Possible to align this with istio release version?

Reply:
done